src/fairseq2/models/wav2vec2/archs.py

```diff
@@ -20,6 +20,30 @@ def _base() -> Wav2Vec2Config:
     return Wav2Vec2Config()
 
 
+# Register the new wav2vec2_arch which returns a Wav2Vec2Config here (which also needs the Wav2Vec2EncoderConfig): 
+# similar to https://github.com/fairinternal/seamless_communication/blob/f3572d6fc77ca6028230dbdbd7611a1b46789949/src/seamless_communication/models/unit_extractor/wav2vec2_layer_output.py#L56
+@wav2vec2_arch("mms_base_300m")
+def _mms_base_300m() -> Wav2Vec2Config:
+    config = _base()
```

Comment:
Can you initialize the config with `_large_lv60k()` and then change whatever necessary? Since the mms base arch corresponds to wav2vec2 large lv60k arch.